### ISSUE TITLE
[Bug] Crash or Compatibility

### ISSUE THREAD

**Issue by Blackfindus** [Traben-0/Entity_Model_Features#42]
I'm using 1.19 with fabric and installed version 0.2.5, which is the only one that curseforge says to work with 1.19. But it doesn't work, fabric says it's not compatible and that i should use 1.19.2

1. Install EMF version 0.2.5 fabric
2. Open Minecraft with 1.19
3. See error

 - Minecraft Version: 1.19, fabric
 - EMF version: 0.2.5
 - Does the issue persist with only EMF installed?

**Comment by Traben-0** [comment]
yeah i need to update that at some point 1.19 support broke and it wasnt noticed for a while

**Comment by Traben-0** [comment]
removed the flag on curseforge that said it would work with 1.19

**Comment by tana-to** [comment]
> removed the flag on curseforge that said it would work with 1.19

is it working now ?

**Comment by Traben-0** [comment]
Whatever the last listed 1.19 version is should work. 
For now I only target 1.19.2 and 1.19.4

**Comment by tana-to** [comment]
I use 1.19.2 but its not working
